src/messages.js

```diff
@@ -427,6 +427,7 @@ function openHelpRequestBlocks() {
                         option('Expert UI', 'xui'),
                         option('Financial Remedy', 'finrem'),
                         option('FPLA'),
+                        option('FPRL'),
```

Comment:
you could add a display name too, up to you? 

```suggestion
                        option('Family Private Law', 'FPRL'),
```